<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextEdit { background-position: bottom center }</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="backgroundLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>13</y>
      <width>1041</width>
      <height>791</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Images/BarPhoto.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="referenceButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>300</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reference</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Reference.jpg</normaloff>:/Images/Reference.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>200</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="teachButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>300</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Learn</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Teach.jpg</normaloff>:/Images/Teach.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>200</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="quizButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>300</width>
      <height>220</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quiz</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Quiz.jpg</normaloff>:/Images/Quiz.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>200</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="bourbonButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>51</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Bourbon</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Bourbon.png</normaloff>:/Images/Bourbon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="campariButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>51</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Campari</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Campari.png</normaloff>:/Images/Campari.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="cognacButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>51</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Cognac</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Cognac.png</normaloff>:/Images/Cognac.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="cointreauButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>61</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Cointreau</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Cointreau.png</normaloff>:/Images/Cointreau.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="vodkaButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>51</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Vodka</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Vodka.png</normaloff>:/Images/Vodka.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="citrusVodkaButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>61</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Citrus
Vodka</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/CitrusVodka.png</normaloff>:/Images/CitrusVodka.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="ryeButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>41</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
QTextEdit { background-position: top center }</string>
    </property>
    <property name="text">
     <string>Rye</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/RyeWhiskey.png</normaloff>:/Images/RyeWhiskey.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="champagneButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>71</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Champagne</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Champagne.png</normaloff>:/Images/Champagne.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="darkRumButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>61</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Dark Rum</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/DarkRum.png</normaloff>:/Images/DarkRum.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="darkTequilaButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>51</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Dark
Tequila</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/DarkTequila.png</normaloff>:/Images/DarkTequila.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="dryVermouthButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>61</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Dry
Vermouth</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/DryVermouth.png</normaloff>:/Images/DryVermouth.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="grandMarnierButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>61</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Grand
Marnier</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/GrandMarnier.png</normaloff>:/Images/GrandMarnier.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="ginButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>41</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Gin</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Gin.png</normaloff>:/Images/Gin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="lightRumButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>41</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Light
Rum</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/LightRum.png</normaloff>:/Images/LightRum.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="midoriButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>41</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Midori</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Midori.png</normaloff>:/Images/Midori.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="peachSchnappsButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>70</y>
      <width>61</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Peach
Schnapps</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/PeachSchnapps.png</normaloff>:/Images/PeachSchnapps.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="silverTequilaButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>90</y>
      <width>61</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Silver
Tequila</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/SilverTequila.png</normaloff>:/Images/SilverTequila.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="sweetVermouthButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>80</y>
      <width>61</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Sweet
Vermouth</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/SweetVermouth.png</normaloff>:/Images/SweetVermouth.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="trippleSecButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>90</y>
      <width>51</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Tripple
Sec</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/TripleSec.png</normaloff>:/Images/TripleSec.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="scotchButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>80</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Scotch</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/ScotchImage.png</normaloff>:/Images/ScotchImage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tumblerButton">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>510</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/TumblerGlass.png</normaloff>:/Images/TumblerGlass.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <zorder>quizButton</zorder>
   <zorder>teachButton</zorder>
   <zorder>referenceButton</zorder>
   <zorder>backgroundLabel</zorder>
   <zorder>campariButton</zorder>
   <zorder>champagneButton</zorder>
   <zorder>cointreauButton</zorder>
   <zorder>vodkaButton</zorder>
   <zorder>darkRumButton</zorder>
   <zorder>bourbonButton</zorder>
   <zorder>ryeButton</zorder>
   <zorder>citrusVodkaButton</zorder>
   <zorder>cognacButton</zorder>
   <zorder>darkTequilaButton</zorder>
   <zorder>dryVermouthButton</zorder>
   <zorder>grandMarnierButton</zorder>
   <zorder>ginButton</zorder>
   <zorder>lightRumButton</zorder>
   <zorder>midoriButton</zorder>
   <zorder>peachSchnappsButton</zorder>
   <zorder>silverTequilaButton</zorder>
   <zorder>sweetVermouthButton</zorder>
   <zorder>trippleSecButton</zorder>
   <zorder>scotchButton</zorder>
   <zorder>tumblerButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1303</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
